### ISSUE TITLE
Update Homepage table to render data from DB

### ISSUE THREAD

**Issue by safchow** [uwblueprint/supportive-housing#70]
## Summary
The log records table on the homepage currently uses mock data. This should be replaced with the log records from the database instead. Some of the backend work is already done in #65. 

## Goals
- [ ] After #65 is merged in, use the `logRecords` state in `LogRecords.tsx` to render the data from the db on the page.
- [ ] There is some investigation involved, as `logRecords.map()` doesn't work, so simply changing `mockRecords.map` isn't the solution.
- [ ] Another note: Date and Time are two different columns on the frontend but they're stored as one column in the db, they should be correctly split into their respective columns when rendered.
